tests/imposter/private-api.yaml

```diff
@@ -21,6 +21,8 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+        "403":
```

Comment:
This needs to be added into the appropriate place in the [API private-api.yml](https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/main/infrastructure/private-api.yaml#L18-L29) file

Reply:
PR has been merged in the API repo